tests/ui/README.md

```diff
@@ -0,0 +1,61 @@
+UI Tests for Add-Ons Discovery Pane (discopane)
+==================================
+
+Thank you for checking out Mozilla's addons-frontend ui test suite.
+This repository contains Selenium tests used to test the discovery pane.
+
+How to run the tests locally
+-----------------------------------------
+
+### Clone the repository
+If you have cloned this project already then you can skip this, otherwise you'll need to clone this repo using Git.
+If you do not know how to clone a GitHub repository, check out this
+[help page][git-clone] from GitHub.
+
+If you think you would like to contribute to the tests by writing or maintaining them in the future,
+it would be a good idea to create a fork of this repository first, and then clone that.
+GitHub also has great instructions for [forking a repository][git-fork].
+
+### Create or activate a Python virtual environment
+You should install this project's dependencies (which is described in the next step) into a virtual environment
+in order to avoid impacting the rest of your system, and to make problem solving easier.
+If you already have a virtual environment for these tests, then you should activate it,
+otherwise you should create a new one.
+For more information on working with virtual environments see our
+[summary][virtualenv].
+
+### Install dependencies
+Install the Python packages that are needed to run our tests using pip. In a terminal,
+from the the project root, issue the following command:
+
+```bash
+$ pip install -Ur requirements.txt
+```
+
+### Run the tests
+Tests are run using the command line. Below are a couple of examples of running the tests:
+
+Before running, ensure the [GeckoDriver executable][geckodriver] (wires) is in your path.
```

Comment:
Might be worth noting it needs to be renamed to wires. Also there's an issue running `0.10.0` not sure if that's this code or an issue in Geckodriver.